lte/gateway/c/core/oai/tasks/amf/amf_app_state_converter.cpp

```diff
@@ -199,13 +199,70 @@ void AmfNasStateConverter::proto_to_ue(
 void AmfNasStateConverter::ue_m5gmm_context_to_proto(
     const ue_m5gmm_context_t* state_ue_m5gmm_context,
     magma::lte::oai::UeContext* ue_context_proto) {
-  // Actual implementation logic will be added as part of upcoming pr
+  ue_context_proto->set_rel_cause(
+      state_ue_m5gmm_context->ue_context_rel_cause.present);
+  ue_context_proto->set_mm_state(state_ue_m5gmm_context->mm_state);
+  ue_context_proto->set_ecm_state(state_ue_m5gmm_context->ecm_state);
+
+  ue_context_proto->set_sctp_assoc_id_key(
+      state_ue_m5gmm_context->sctp_assoc_id_key);
+  ue_context_proto->set_enb_ue_s1ap_id(state_ue_m5gmm_context->gnb_ue_ngap_id);
+  ue_context_proto->set_enb_s1ap_id_key(
```

Comment:
Same comment as above. Please add a field `gnb_ngap_id_key` in the UE Context proto, instead of repurposing the `enb_s1ap_id_key`

Reply:
Addressed review comment.